tests/elm-package.json

```diff
@@ -1,13 +1,14 @@
 {
     "version": "1.0.0",
     "summary": "Sample Elm Test",
-    "repository": "https://github.com/user/project.git",
+    "repository": "https://github.com/ktonon/aws-sdk-elm.git",
```

Comment:
This has to be set to the official project name so it lines up with the native module.